### PR TITLE
stricten settings for key on iOS

### PR DESCRIPTION
**Pull request by berendsliedrecht**
Signed-off-by: Berend Sliedrecht <sliedrecht@berend.io>

### CODE REVIEW

File: examples/mobile_tests/src/lib.rs [hunk 1]
```diff
@@ -96,41 +91,6 @@ fn test_get_by_id_and_sign() {
...
-    let public_key = key.get_public_key().unwrap();
-    let msg  = b"Hello World!";
-
-    let signature = key.sign(msg).unwrap();
-
-    let keypair = P256KeyPair::from_public_bytes(&public_key).unwrap();
-
-    let is_valid = keypair.verify_signature(msg, &signature);
-
-    assert!(is_valid);
```
Comment (TimoGlastra) [inline examples/mobile_tests/src/lib.rs:134]: I think we should keep an option to lower the security settings so we can have a test that does signing + verification. As now we basically have now tests for this.

Comment (TimoGlastra) [reply examples/mobile_tests/src/lib.rs:134]: And we should probably also have some negative tests. Wrong signature / changed content should give an error

**Review by TimoGlastra — CHANGES_REQUESTED** [review]

File: examples/mobile_tests/src/lib.rs (hunk 1 above)
Comment (berendsliedrecht) [reply examples/mobile_tests/src/lib.rs:134]: > I think we should keep an option to lower the security settings so we can have a test that does signing + verification. As now we basically have now tests for this. we decided not to make it optional, right? Or just in askar?

Comment (berendsliedrecht) [reply examples/mobile_tests/src/lib.rs:134]: > And we should probably also have some negative tests. Wrong signature / changed content should give an error

this library does not validate the signature, thats from askar.

**Comment by berendsliedrecht** [comment]
@TimoGlastra added optional lock behind biometrics

**Review by TimoGlastra — APPROVED** [review]

**Pull request merged**